identity/backend/src/main/java/io/camunda/identity/usermanagement/User.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.identity.usermanagement;
+
+public record User(String username, String password, boolean enabled) {
```

Comment:
❓ Do we have a usecase for `enabled`? I wasn't sure if its something that Spring is expecting but IIRC up until now we don't have enabled/disabled in the Identity UI